src/pages/Courses/components/CoursesContainer/CoursesContainer.css

```diff
@@ -0,0 +1,11 @@
+.courses-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(16rem, 1fr));
+  position: sticky;
+  top: 64px;
```

Comment:
Suggestion: I think position sticky is not required here. Code should work without it. Do check.

Reply:
Removed unused css